rizwan171/trello-clone
# Tc-122 attachment

Test Scenarios:
- if you can add a file attachment
- if you can add multiple files
- check if png images have a small preview square image on the card modal when added
- check if other files have the corresponding file extension on the preview thumbnail
- check if clicking the attachment prompts a download with the name of the attachment
- check if you can edit the name of the attachment 
- check after editing the name of the attachment, clicking that attachment will pop up a download with the edited name
- check if you can delete an attachment
- check if you refresh the page the attachments persist

To be address in later tickets:
-Need to do drop file on card modal 
- dominant colour background for image
- Image preview only matches for png file extensions
- Attachments on the card model should be movable and interactive like cards/
- Add image preview page
- set cover photo with the attachment
- able to set cover photo
- Minor design inconsistencies
- clicking an image leads to download pop up instead of image preview page.
- make the card modal more
